test/config_test.rb

```
@@ -135,6 +141,12 @@ def test_time
 
     set_env 'T', '2000-2-2T11:11'
     assert_equal(Time.new(2000,2,2,11,11), Config.time(:t))
+
+    set_env 'T', nil
+    assert_equal(nil, Config.time(:t))
+
+    set_env 'T', 'derp'
+    assert_equal(false, Config.time(:t))
```

this should raise an error too